Konscious.Security.Cryptography.Argon2/LittleEndianActiveStream.cs

```diff
@@ -48,6 +49,12 @@ public void Expose(byte[] data)
             }
         }
 
+        public void Expose(ReadOnlySpan<ulong> span)
```

Comment:
I think everywhere this is used now it can use memory so we could just toss that in here which would mean we didn't have to .ToArray and get an extra copy here.

Reply:
Yeah the issue is you can't capture ref structs in lambdas so there needs to be a Memory<T> here to convert in

Reply:
Ahh that makes sense

Reply:
resolved (since I can't do it myself... thanks GH)